mne_bids/config.py

```diff
@@ -2,6 +2,13 @@
 from mne import io
 from mne.io.constants import FIFF
 
+try:
+    from mne.io import read_raw_persyst
+except ImportError:
+    print('Reading Persyst files requires latest mne dev '
+          'version or 0.21 release.')
+    read_raw_persyst = None
```

Comment:
I would expect somewhere a if read_raw_persyst is not None but I don't see it.

Reply:
Migrated to a private function to do this now.